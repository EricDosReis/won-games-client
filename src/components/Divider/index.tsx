import styled, { css } from 'styled-components';

const Divider = styled.hr`
  ${({ theme }) => css`
    background: rgba(181, 181, 181, 0.3);
    border: 0;
    height: 0.1rem;
    margin: ${theme.spacings.xlarge} auto;
  `}
`;

export default Divider;
