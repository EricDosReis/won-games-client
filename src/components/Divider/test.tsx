import { renderWithTheme } from 'utils/tests/helpers';

import Divider from '.';

describe('<Divider />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Divider />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
