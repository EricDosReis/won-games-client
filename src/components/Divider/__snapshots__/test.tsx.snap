// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should render the heading 1`] = `
.c0 {
  background: rgba(181,181,181,0.3);
  border: 0;
  height: 0.1rem;
  margin: 4.8rem auto;
}

<hr
  class="c0"
/>
`;
